### PR TITLE
Updated documentation on MS MARCO v2 doc (segmented) collection

### PR DESCRIPTION
**Pull request by lintool**

### CODE REVIEW

File: docs/experiments-msmarco-v2.md [hunk 1]
```diff
@@ -164,18 +164,36 @@ recall_100            	all	0.5970
...
+
+We segmented the document collection with [this Python script](https://github.com/castorini/pyserini/blob/master/scripts/msmarco_v2/segment_docs.py).
+As a summary, the script trims each document to 10k characters, and then applies a 10-sentence sliding window with a 5-sentence stride to generate the passages.
+This approach is similar to the results reported in [Pradeep et al. (2021)](https://arxiv.org/abs/2101.05667), which has been demonstrated to be effective.
+Sentence chunking is performed with spaCy (v2.3.5); the version is important if you want to _exactly_ reproduce our results from scratch with the Python script above.
+We have also experimented with _not_ trimming each document to the first 10k characters; the corpus becomes much bigger and the results become worse on the dev queries below.
+
+For convenience, the Microsoft organizers have kindly offered to host of copy of the segmented corpus, `msmarco_v2_doc_segmented.tar` (26 GB, MD5 checksum of `f18c3a75eb3426efeb6040dca3e885dc`).
+The tarball can be downloaded [here](https://msmarco.blob.core.windows.net//msmarcoranking/msmarco_v2_doc_segmented.tar).
```
Comment (crystina-z) [inline docs/experiments-msmarco-v2.md:180]: the extra / here seems to make the link broken?

Comment (lintool) [reply docs/experiments-msmarco-v2.md]: Thanks for catching!

**Review by jacklin64 — APPROVED** [review]
Look nice!

**Review by justram — APPROVED** [review]
LGTM

**Review by crystina-z — APPROVED** [review]

**Pull request merged**